### PR TITLE
add helper sheets

### PR DESCRIPTION
**Pull request by ozym**
@nbalfour A first go at some cheat-sheets for some common tasks. It's more of a framework at the moment and will need fleshing out (or rewriting).

### CODE REVIEW

**Comment by elidana** [comment]
can't resist but looking around here. WOW!!! thanks Mark!

**Comment by elidana** [comment]
Hi @ozym , 
I've been looking at the GNSS.md helper sheet and added a (small) bunch of missing things.
I like it a lot, you did an awesome job in considering everything (delta prerequisite, field prerequisites, operator checks, all of it)!

I've only added a few missing bits, and the version I've slightly modified can be found here:
https://github.com/elidana/delta/blob/process-gnss/sheets/GNSS.md

If you're happy with that I can push that file to this pr.

But....as a general comment for the GNSS markdown,  I'd like to merge some of the common tasks to help the person that is doing the metadata change in reviewing/updating all the necessary files for a given task.
For example, if a mark need to be created/closed, one can easily miss a necessary file and don't update all the required files. 
I'm working (still in progress) on a slightly different version of your markdown that can be found here
https://github.com/elidana/delta/blob/helpsheet-gnss/sheets/GNSS.md 

Not major changes there, I'm only grouping common tasks based on different scenarios.
But before going too much further, I'd like to see what you (and the other) think.

File: sheets/ASSETS.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+## :two: _Adding a new asset entry_
+
+Find the appropriate __asset__ file for the type of equipment.
+
+> ### :information_source: General requirements
+> 
+> * The make, model, and serial number of the new `assets.
+> * Any optional asset "numbers" that may be available.
+> * Any extra equipment notes that may be appropriate.
+>
```
Comment (elidana) [inline sheets/ASSETS.md]: Hi @ozym , can we please add something here specific to new GNSS equipment?
Something like:
```
for GNSS equipment, the name used in the "_model_" field need to be present in the IGS receiver and antenna table file: ftp://igs.org/pub/station/general/rcvr_ant.tab 
```

Comment (ozym) [reply sheets/ASSETS.md]: That looks great, either I merge this into this PR or wait and merge it into master after this one has been sorted.

Comment (ozym) [reply sheets/ASSETS.md]: It may be that these sheets don't live in this repo.

Comment (elidana) [reply sheets/ASSETS.md]: thanks!!! whatever works best for you, I'll keep that branch and wait to see when this PR is sorted. 
You did a great job with these helpsheets, thanks heaps!

Comment (elidana) [reply sheets/ASSETS.md]: relatively to the assets, that's ok. 
We will only need to make sure that the "model" names we use are IGS-compliant. 
I can add this note in the "GNSS.md" overall steps as well.

**Comment by ozym** [comment]
@nbalfour have moved into docs, will need another review tick.

**Pull request merged**